Firebase/InstanceID/FIRInstanceID.m

```diff
@@ -639,41 +638,41 @@ + (void)load {
 + (nonnull NSArray<FIRComponent *> *)componentsToRegister {
   FIRComponentCreationBlock creationBlock =
       ^id _Nullable(FIRComponentContainer *container, BOOL *isCacheable) {
+    // InstanceID only works with the default app.
+    if (!container.app.isDefaultApp) {
+      // Only configure for the default FIRApp.
+      FIRInstanceIDLoggerDebug(kFIRInstanceIDMessageCodeFIRApp002,
```

Comment:
Shouldn't this be a warning or even an error?

Feel free to ignore--I see that this is just copypasta from the prior state.

Reply:
This is also a "this code should never be run" so maybe it should be an error, but I'll leave it as is since it was copied from before.